### ISSUE TITLE
Use HTTPS connection instead of HTTP

### ISSUE THREAD

**Issue by Dom1n1c** [CFenner/MagicMirror-LocalTransport-Module#12]
i noticed that the train images will be shown as "insecure"

**Comment by CFenner** [comment]
You're right, I used hardcoded http [here](https://github.com/CFenner/MagicMirror-LocalTransport-Module/blob/master/localtransport.js#L141). Can you test, if the icon is also available via https? I am not at home right now and my mirror is currently not in use :)

**Comment by Dom1n1c** [comment]
Google maps works with https 👍 
I need to proxy the weather api (supports http only), but i informed the author of this module about it.
<img width="405" alt="smartmirror_https" src="https://cloud.githubusercontent.com/assets/8407566/21472276/458396f8-cad1-11e6-801d-45e6d7aeb11d.png">
